### PR TITLE
C2LC-391: Fix AddNode button's svg gets shifted when focused

### PR DESCRIPTION
**Pull request by chosww**
See [C2LC-391](https://issues.fluidproject.org/browse/C2LC-391) for details.

### CODE REVIEW

File: src/Themes.scss [hunk 1]
```diff
@@ -242,7 +242,7 @@ body.light-theme {
 
     button.btn.AddNode__expanded-button {
         background-color: white;
-        border: 3px solid #505862;
+        border: 2px solid #505862;
```
Comment (the-t-in-rtf) [inline src/Themes.scss:245]: Why not use the `addNodeExpandedButtonBorderWidth` variable here?

**Comment by the-t-in-rtf** [comment]
This addresses the issue across all themes.  I had one minor comment on a hard-coded width that seems to match an existing variable.

File: src/Themes.scss (hunk 1 above)
Comment (chosww) [reply src/Themes.scss:245]: Currently, addNodeExpandedButtonBorderWidth is defined in AddNode.scss and from the theming work, any border for the expanded add node button are hardcoded in Theme.scss.

Comment (chosww) [reply src/Themes.scss:245]: I am thinking, we should merge this PR with this change, and have a clean up task to move any commonly used values in different themes to variables.scss, what do you think?

Comment (the-t-in-rtf) [reply src/Themes.scss:245]: That sounds reasonable.  We could also consider making some more animations into mixins.

Comment (chosww) [reply src/Themes.scss:245]: This work is ready for a review @sbates-idrc, please let me know what do you think about the comments Tony and I left.

**Pull request merged**